docs/03-extensions/payzen/how-to/magento2.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Use as a Magento2 payment
+description:
+  "This guides explains how to configure Payzen payment with Magento 2 module."
+sidebar_position: 4
+---
+
+<p>{frontMatter.description}</p>
+
+:::caution Information
+
+<span>
+  This integration is aimed at being transparent for administrators and
+  developers. That is why we haven't duplicated documentation from existing
+  Magento resources. Please <ContactLink /> if you need further assistance.
+</span>
+
+:::
+
+Front-Commerce Magento2 module contains headless payment adapters for the
+[**PayZen Standard** method](https://github.com/lyra/plugin-magento) (Lyra's
+official Magento module).
+
+The PayZen module must be configured in a normal way, as for a non-headless
+Magento store.
+
+## Pre-requisite
+
+:::danger
+
+For the time being, we only support PayZen accounts using a SHA1 signature. If
+your account uses a SHA-256 signature, please contact support to learn about the
+modifications required in the Front-Commerce Module to make it compatible.
+
+:::
+
+- [Payzen Account](https://www.payzen.eu/)
+- [Payzen Module installed in Magento 2](#installation)
+
+## Installation
+
+:::danger PHP Module Compatibility
+
+At the time of writing, only version 2.5.13 of the Payzen module is supported.
```

Comment:
**nitpick:** we do support previous versions and likely other patch versions (in case they're released)

**suggestion:**  rephrase
```suggestion
At the time of writing, we support the Payzen module up to version 2.5.
```

Reply:
fixed with your suggestion and added to 2.x docs